libs/design/src/molecules/modal/animations/modal-animation.ts

```diff
@@ -1,23 +1,18 @@
 import {
-  animate,
-  state,
-  style,
-  transition,
-  trigger,
-  AnimationTriggerMetadata,
+	animate,
+	state,
+	style,
+	transition,
+	trigger,
+	AnimationTriggerMetadata,
 } from '@angular/animations';
 
 export const daffFadeAnimations: {
-  readonly fade: AnimationTriggerMetadata,
+	readonly fade: AnimationTriggerMetadata;
 } = {
-  fade: trigger('fade', [
-    state('show', style({
-      opacity: '1',
-    })),
-    state('void',style({
-      opacity: 0
-    })),
-    transition('void <=> show',
-        animate('350ms cubic-bezier(0.25, 0.8, 0.25, 1)'))
-  ])
+	fade: trigger('fade', [
+		state('open', style({ opacity: 1, })),
+		state('closed', style({ opacity: 0, transform: 'translateY(-20px)' })),
+		transition('open <=> closed', animate('100ms cubic-bezier(0.25, 0.8, 0.25, 1)')),
```

Comment:
Modal timing differs from backdrop. I think they should be the same. It looks awkward to have backdrop fade out later than the modal.